Dockerfile

```diff
@@ -37,6 +37,9 @@ USER 1001:1001
 
 COPY --from=0 /go/bin/dex /usr/local/bin/dex
 
+# Copy module dependencies for CVE scanning / dependency analysis.
+COPY go.mod go.sum /dex-dependencies/
```

Comment:
I'd prefer `/opt/dex` or something similar. Also, you'll probably need the module files for the api module.

Reply:
@sagikazarmark okay done:

```
$ docker run --entrypoint sh quay.io/dexidp/dex:latest -c 'find /opt/'
/opt/
/opt/dex
/opt/dex/dependencies
/opt/dex/dependencies/go.sum
/opt/dex/dependencies/go.mod
/opt/dex/dependencies/api
/opt/dex/dependencies/api/v2
/opt/dex/dependencies/api/v2/go.sum
/opt/dex/dependencies/api/v2/go.mod
```